sim/hunter/wyvern_strike.go

```diff
@@ -42,6 +42,7 @@ func (hunter *Hunter) getWyvernStrikeConfig(rank int) core.SpellConfig {
 			return hunter.DistanceFromTarget <= 5
 		},
 
+		CritDamageBonus: hunter.mortalShots(),
 		DamageMultiplier: 1,
```

Comment:
I missed this before but Wyvern Strike is also missing a threat multiplier. I'm not sure about Wing Clip above? If it deals damage now it probably needs one too

Reply:
Added